src/riak_kv_qry_compiler.erl

```diff
@@ -113,6 +128,14 @@ prepend_select_columns([_|_] = MultiCols, Acc) ->
 prepend_select_columns(V, Acc) ->
     [V | Acc].
 
+%% copy pasta mapfoldl from lists.erl, otherwise this reports
+%% a warning in dialyzer!
+my_mapfoldl(F, Accu0, [Hd|Tail]) ->
```

Comment:
Really?  You can't use `lists:mapfoldl`? This is the dialyzer bug mentioned the other day?